Vagrantfile

```diff
@@ -18,17 +18,90 @@ end
 ENV['ANSIBLE_CONFIG'] = ansible_config_file
 
 NMONS        = ENV["VMS"] || settings['vms']
-SUBNET       = settings['subnet']
 BOX          = settings['vagrant_box']
 BOX_VERSION  = settings['box_version']
 memory       = settings['memory']
 
+SUBNET_PREFIX          = settings['subnet_prefix']
+SUBNET_ASSIGNMENT_DIR  = "/tmp/volplugin_vagrant_subnets/"
+SUBNET_ASSIGNMENT_FILE = "subnet_assignment.state"
```

Comment:
this file is getting created in the wrong directory sometimes. Please use the pattern on line 9 for calculating the directory.